config/initializers/spree.rb

```diff
@@ -16,3 +16,41 @@
 end
 
 Spree.user_class = "Spree::User"
+
+attachment_config = {
+  styles: {
+    mini:     "48x48>",
+    small:    "100x100>",
+    product:  "240x240>",
+    large:    "600x600>"
+
+  },
+  path:           "/:class/:id/:style/:basename.:extension",
+  default_url:    "/:class/:id/:style/:basename.:extension",
+  default_style:  "product"
+}
+
+if Rails.env.production?
+  # S3 image upload settings
+  attachment_config.merge! ({
+                             s3_credentials: {
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.